openedx/core/lib/edx_api_utils.py

```diff
@@ -45,14 +42,7 @@ def get_edx_api_data(api_config, user, resource, resource_id=None, querystring=N
             return cached
 
     try:
-        jwt = get_id_token(user, api_config.OAUTH2_CLIENT_NAME)
```

Comment:
@ahsan-ul-haq I think it's better to use the keyword argument for now. Our goal is to standardize inter-IDA communication.